manifests/e/EmoteInteractive/RemoteMouse/4.503/EmoteInteractive.RemoteMouse.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Automatically updated by the winget bot at 2023/Jan/01
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: EmoteInteractive.RemoteMouse
+PackageVersion: "4.503"
+PackageLocale: en-US
+Publisher: Remote Mouse
+PackageName: Remote Mouse
+PackageUrl: https://www.remotemouse.net
+License: Copyright (c) Remote Mouse

```

```suggestion
License: Proprietary
```
A copyright isn't a license